tests/setup.py

```diff
@@ -0,0 +1,13 @@
+from flumes.config import Config
```

Comment:
A few general comments to help us structure the code better and be prepared for the long run.

- General best practice for the name of this file is `conftest.py `to diffentiate it from `setup.py `that might be related to the flumes package as a whole. `conftest.py` will be loaded automatically by pytest when you execute it at the path where `conftest.py` is located.
- We will define fixtures with "global" scope in this file so we will need to `import pytest`